tests/health_checks/NDv4/test-bandwidth.sh

```diff
@@ -35,20 +35,259 @@ catch_error() {
 	fi
 }
 
+create_benchmark() {
```

Comment:
If we can download the benchmarks from superbench repo, why would we re-write it here?
```
#string="https://raw.githubusercontent.com/microsoft/superbenchmark/release/"\
#	"0.5/superbench/benchmarks/micro_benchmarks/gpu_copy_performance/"\
#	"gpu_copy.cu"
```

Reply:
We have written it here so the health checks can be used even without access to the external internet. Sometimes that may be blocked after machine setup. But that's a good point I should get rid of that comment.

Reply:
How about we save gpu_copy.cu directly? Creating a .cu file via Bash (EOF) seems weird to me.

Reply:
Yeah, we can certainly do that. I was trying to make each test self-contained, but if that would be clearer, I can move the gpu copy test into its own file.

Reply:
Alright I have separated the gpu copy cuda code out to its own file ("gpu-copy.cu"). 

Now i test if the gpu-copy executable exists already. If it does I run it. If the executable does not already exist I test if gpu-copy.cu exists. If the cu file does not exist the test fails with an error message saying it cannot find gpu-copy.cu. If it does exist it is compiled.